### PR TITLE
Save images from docs in repo

### PR DESCRIPTION
**Pull request by Petrprogs**
All images (seems) moved to "resources" folder

### CODE REVIEW

File: docs/resources/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This directory contains resources (images) used in docs
```
Comment (nevack) [inline docs/resources/README.md:1]: Is this file really needed? I think this directory name is self-explanatory :)

Comment (Petrprogs) [reply docs/resources/README.md:1]: Github dont allow to create empty directory from web interface

Comment (ckerr) [reply docs/resources/README.md:1]: But it's no longer an empty directory, so maybe it could be removed now?

Comment (Petrprogs) [reply docs/resources/README.md:1]: Ok!

Comment (Petrprogs) [reply docs/resources/README.md]: Done!

Comment (Petrprogs) [reply docs/resources/README.md]: @nevack, solved?
